otter/integration/lib/autoscale.py

```diff
@@ -132,6 +140,79 @@ def set_launch_config(self, rcs, launch_config):
             .addCallback(lambda _: rcs)
         )
 
+    def get_group_config(self, rcs):
+        """
+        Returns the group configuration recorded in the TestResources object.
+
+        :param TestResources rcs: The integration test resources instance.
+            This provides useful information to complete the request, like
+            which endpoint to use to make the API request.
+
+        :return:
+        """
+        # Retrieve the configuration from the stored response to group
+        # creation since that will include non-provided values
+        config = [g['group']['groupConfiguration'] for g in rcs.groups
+                  if g['group']['id'] == self.group_id]
+        # Since this should always provide a match for a valid scaling group,
+        # allow the exception to be raised if nothing is found.
+        if verbosity > 0:
+            print('ScalingGroup.get_group_config will return: ')
+            pp.pprint(config[0])
+        return config[0]
+
+    def update_group_config(self, rcs, name=None, cooldown=None,
+                            minEntities=None, maxEntities=None, metadata=None,
+                            replacement_config=None):
+        """
+        Update the group configuration of a scaling group. Ifa replacement
+        configuration is provide, it will be used to replace all values.
+        Otherwise, only the specified parameters will be altered.
+        """
+        # Get the old config
+        old_config = self.get_group_config(rcs)
+        new_config = {}
+        if replacement_config is not None:
```

Comment:
Non-blocking - probably better to eliminate the replacement_config, since this can be easily synthesized from all the others, and it's less confusing if function parameters didn't have an order of precedence/didn't supercede each other.